middleware/HuaweiCloud-Kafka-autofill.yml

```diff
@@ -302,6 +302,171 @@ serviceActions:
           isDataLost: true
           isServiceInterrupted: true
         isReadOnly: false
+serviceObjects:
```

Comment:
actually the same service objects which i gave you are for database. So please move these samples to database service.